composer.json

```diff
@@ -2,12 +2,12 @@
     "name": "shopgate/cart-integration-magento2-bolt",
     "description": "Shopgate bolt module for Magento 2, Extends check_cart functionality to make it possible to prepare preserve and return information about a quote",
     "type": "magento2-module",
-    "version": "0.2.6",
+    "version": "0.3.0",
     "license": "Apache-2.0",
     "require": {
         "shopgate/cart-integration-magento2-base": "^2.9.17",
         "shopgate/cart-integration-magento2-export": "^2.9.15",
-        "boltpay/bolt-magento2": "~2.0"
+        "boltpay/bolt-magento2": "^2.4"
```

Comment:
We should also mention in changelog that the support was dropped for version below 2.4

Reply:
added line to change log to make sure it is understood that versions lower than 2.4 of the bolt pay extension are no longer supported.